### DISCUSSION TITLE
2023-07-06

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#10] [daily_meeting]
### 개인 일정 종합
마코 : 테코톡 진행 근로 1시 30분 ~ 3시
백엔드 : 오전 강의 진행

#### Keep
중간에 잡담 타임을 가지는 것이 좋았다.
점심 한식뷔페 먹은 게 좋았습니다. 가끔 한번씩 가도 좋을거같다!
오류를 해결하기 위한 과정은 항상 가져갔으면 한다!
각자 업무를 분담해서 병렬적으로 일을 하는 것이 좋았다.
액션 아이템을 적재적소에 잘 쓰고 있다.

#### Problem
- 회의록 관련
  - 전체 회의가 갈 수록 줄어들텐데 회의록을 어떻게 할 지 정해야 될 것 같다.
  - 회의록에 어떤 것을 적을지? 다함께 의논할 내용에 대해서만 다룰지?가 명확하지 않다. 개인적인 의견은 스프린트를 시작할 때, 중요한 전체 안건이 있을 때, 적으면 어떨까?
  - 토론하는 동안 나오는 이야기들을 기록해야 할 필요성이 있는 것 같다. 자꾸 어디까지 이야기했었는지 까먹게 된다.
  - 논의과정에서 발생한 문제상황과 이를 해결하기 위해 모색한 과정들이 잘 기록되고 있는지 의문이 든다.

- 의사소통 관련
  - 회의를 할 때 그냥 각자 의견만 주장하는 느낌이 든다. 그러다 보니 서로가 한 이야기를 제대로 이해를 하지 못하는 경우가 있다. 정말 좋은 의견인데 묻혀 넘어가버린다면? 과연 우리 끼리도 이해하지 못했는데 다른 사람이 들었을 때 이해할 수 있을까?

- 태스크 관리 관련
  - 할 일 목록이 실시간으로 정리할 곳이 없다.

#### Try
우선 서로의 의견에 대해 주의 깊게 듣는 태도가 필요할 것 같다. 그리고 나서 이해가 안간다면 얼마든지 되물어 볼 수 있어야 하지 않을까?
회의하다가 공동으로 한 중요한 의사결정이 있다면, 기록해두고 넘어가기? 세이브포인트처럼.
먼저 할 일 목록 관리 페이지를 만들어뒀다가 프로젝트 사용이 가능해지면 이슈로 옮긴다.
문제상황과 모색한 해결방안, 해결방안 선정 이유를 최대한 회의록에 정리하고 이후 정제해서 완성된 글을 포스팅한다.

### 오늘의 할 일
- 공통
  - 발표 준비
  - 그라운드 룰 wiki 고도화 및 팀 문화 정리
  - 로고 및 서비스 메인 색상 얘기해보기
  - github Project 칸반보드 정리
  - 기술블로그 세팅
- 백엔드
  - 기술문서 마무리(자바 뭐쓸지, 스프링 뭐쓸지)
- 프론트엔드
  - 와이어 프레임 제작

### 액션아이템 선정
- 프로젝트와 관련된 Task들을 ToDo 태그를 만들어서 이슈로 정리하기!
- 새로운 의견 제시 전 이전 의견에 대한 이해 여부를 충분히 표현하고 넘어가기
- 반대 의견 제시 시 상대방의 의견을 한 번 정리하고 자신의 의견을 개진하는 방식으로 말하기
- 구두로 논의하는 과정에서 디스커션을 열고 각자의 의견 및 근거를 댓글로 정리하는 방식으로 회의 진행하기

**Comment by MoonJeWoong** [comment]
### 건의사항
KPT 회고에서 처음에는 팀 문화나 액션 아이템이 정해진게 없어서 의미가 컸다.
하지만 항상 액션아이템을 뽑아내는 건 현실적으로 힘들다고 생각한다.
좀 더 의미있는 활동을 위해 다음을 제안한다.
- 체크아웃 회고 시간에 오늘했던 일들 중 정리해놓을 만한 사항을 공유하고 정리해두기
- 그냥 정리하자고 하면 또 방향을 잃을 가능성이 크니 다음과 같은 양식을 정해두면 어떨지 제안.

정리할 양식
- 발생한 문제상황
- 해결하기 위해 시도해본 것들
  - 어떤 점이 좋았는지
  - 어떤 점이 아쉬웠는지
- 결론
  - 앞에서 논의한 해결방안들 중 어떤 것을 선택했는지
  - 처음의 문제상황을 어떻게 개선시켰는지
  - 얼만큼 개선시켰는지 (구체적인 수치가 있다면 같이 기록)
---

### 체크인
9점 - 오늘 개운하게 일어났고 생각보다 날씨도 많이 습하지 않아서 기분이 좋습니다!

### 체크아웃
10점 - 발표준비를 하면서 자잘한 task를 많이 처리한 것 같아서 좋습니다. 우리 organization + 기술 블로그 배포한 것도 너무 좋아요!

**Comment by nlom0218** [comment]
### 체크인

9점 - 혓바늘이 나서 신경쓰인다ㅠㅠ

### 체크아웃

10점 - 피그마를 처음 다루어봤는데, 너무 재밌었다.

**Reply by woosung1223** [reply]
알보칠이 직빵인디

**Comment by jaehee329** [comment]
### 체크인

8점 - 약간의 만성 피로, 어떻게 해결해야 할까요? 오늘도 달려봅시다요

### 체크아웃

6점 - 혓바닥이 데여서 불편하네요

**Comment by woo-jk** [comment]
### 체크인
8점 - 이번주 중에 제일 피곤한 아침 ㅠ^ㅠ

### 체크아웃

10점 - 와이어프레임을 완성했다.

**Comment by aak2075** [comment]
### 체크인

9점 - 피곤했으나 노아의 비타민으로 충전💪 오늘 하루도 파이팅

### 체크아웃

10점 - 발표 준비 마무리돼서 기분 좋다. 집가서 맛난거 무야지

**Comment by woosung1223** [comment]
### 체크인
7점 - 경제적 이슈로 인해 라떼를 사지 못했습니다..

### 체크아웃
7점 - 할일은 어느정도 마친거같아 좋습니다 하지만 배고프기에 마이나스 삼점

**Comment by yeopto** [comment]
### 체크인

6점 - 너무 피곤합니다. 고민이 많아졌습니다.

### 체크아웃

10점 - 마코와 함께 1차 데모 PPT를 너무 잘 만들었다. 뿌듯해서 10점
